span_sink.go

```diff
@@ -398,10 +407,91 @@ func (b *blackholeSpanSink) Start(*trace.Client) error {
 	return nil
 }
 
-func (b *blackholeSpanSink) Ingest(ssf.SSFSpan) error {
+func (b *blackholeSpanSink) Ingest(*ssf.SSFSpan) error {
 	return nil
 }
 
 func (b *blackholeSpanSink) Flush() {
 	return
 }
+
+type metricExtractionSink struct {
```

Comment:
I think we need a rebase here that will move this into something like `sinks/spanmetrics/spanmetrics.go` or whatever you wanna name it!